docs/src/components/productX/XChartsDemo.tsx

```diff
@@ -77,24 +77,25 @@ export default function XChartsDemo() {
           }}
         >
           <Typography variant="body2" fontWeight="bold" sx={{ mr: 1 }}>
-            Experiment with Charts now!
+            Start using the Charts now!
           </Typography>
           <Chip
             variant="outlined"
-            label="Alpha"
-            color="warning"
+            label="Available now"
+            color="success"
             size="small"
             sx={(theme) => ({
               pb: 0.2,
               fontWeight: theme.typography.fontWeightSemiBold,
-              color: (theme.vars || theme).palette.warning[300],
-              borderColor: alpha(theme.palette.warning[300], 0.3),
-              background: alpha(theme.palette.warning[800], 0.3),
+              color: (theme.vars || theme).palette.success[300],
+              borderColor: alpha(theme.palette.success[300], 0.3),
+              background: alpha(theme.palette.success[800], 0.3),
             })}
           />
         </Box>
         <Typography variant="body2" color="text.secondary" sx={{ mb: 2 }}>
-          Using D3.js for data manipulation and SVG for rendering, Charts are coming to MUI X!
+          Using D3.js for data manipulation and SVG for rendering, Charts are now available on MUI
+          X!
```

Comment:
I think it would be best to use a non-breaking space here

```suggestion
          Using D3.js for data manipulation and SVG for rendering, Charts are now available on MUI X!
```

to avoid: 

![Screenshot 2023-11-25 at 12 21 54](https://github.com/mui/material-ui/assets/3165635/a0f2edcc-d435-4bec-b916-3025a2683b79)